go.mod

```diff
@@ -3,16 +3,16 @@ module github.com/onflow/flow-go-sdk
 go 1.13
 
 require (
-	cloud.google.com/go v0.97.0 // indirect
 	cloud.google.com/go/kms v1.0.0
-	github.com/ethereum/go-ethereum v1.9.9
-	github.com/onflow/cadence v0.20.1
-	github.com/onflow/flow-go/crypto v0.24.3
-	github.com/onflow/flow/protobuf/go/flow v0.2.2
+	github.com/ethereum/go-ethereum v1.9.13
+	github.com/onflow/cadence v0.21.2
+	github.com/onflow/flow-go v0.24.6
```

Comment:
I think we should avoid including `onflow/flow-go` as a dependency in `flow-go-sdk` because it will add a lot of bloat if we include the entire node software.

In this case, I would even say its OK to copy the implementation of `flow.EventsMerkleRootHash` to avoid having to import it.

Reply:
Ah, looks like it uses the Merkle tree package. Maybe that could be made into its own module?

Reply:
Yes, I remember having that exact conversation with Ramtin and we concluded that if it will work it's ok but he said is else willing to extract it somewhere. cc @ramtinms how should we proceed? I btw agree with this, but at the same time I also think we should also remove whole ethereum node module just for sake of RLP, so since that is now included it begs to ask how important this is, I remember it was required by some integration (not sure I can say which) and how long it will take to extract. Maybe Ramtin should say.